docs/release-guide.md

```diff
@@ -12,20 +12,59 @@ please list the main repository for those in this section:
 
 * Rakudo https://github.com/rakudo/rakudo
 
-## Preparation
+## Terminology
+
+The following terms are used in this document and in other related papers:
+
+- _language revision_ or _revision_
+
+  A set of specifications defining a Perl 6 language variant. Revisions are represented by symbolic names consisting of
+  a single letter of Latin alphabet and may have an alias. For example, revision `c` is `Christmas`; `d` is `Divali`.
```

Comment:
Maybe we should drop this? So that `c` is just `c`. I remember a lot of bikeshedding and it'd be nice to avoid it.

Reply:
c, indeed, *was* Christmas, and d *Diwali* (with w). It says "may" have an alias. Don't see any major problem with this.

Reply:
@JJ I have removed it anyway. Basically, the document is purely technical and it adds nothing useful to have those aliases in there.